src/app/issue-detail/comment/comment.component.ts

```diff
@@ -54,4 +56,22 @@ export class CommentComponent implements OnInit, OnDestroy {
     });
   }
 
+  /**
+   * Deletes the current comment.
+   */
+  public deleteComment(): void {
+    // Alert for confirmation
+    const confirmed = confirm('Are you sure you want to delete this?');
```

Comment:
Use RemoveDialogComponent instead

Reply:
Too much effort for user to delete a comment with RemoveDialogComponent. Github is also using the implemented dialog for comment deletion.